README.md

```diff
@@ -35,9 +35,18 @@ artifacts.first.url
 # => "http://opscode-omnibus-packages-current.s3.amazonaws.com/mac_os_x/10.9/x86_64/chef-12.5.1%2B20151009083009-1.dmg"
 ```
 
+## Unstable channel
+The `:unstable` channel is currently only available when connected to Chef's internal network.
+Configure Artifactory access by setting the following environment variables:
+```
+# default endpoint
+# export ARTIFACTORY_ENDPOINT='http://artifactory.chef.co'
```

Comment:
I would remove this line :smile: to keep things simple. We can help put the one who wants to speak to a different artifactory instance...